### ISSUE TITLE
Not working with Cinnamon 4.3.0

### ISSUE THREAD

**Issue by Lantesh** [germanfr/cinnamon-transparent-panels#35]
```
 * Extension version: 4.0~20181106 (2019-10-27 10:10:23)
 * Cinnamon version: 4.3.0
```

[xsession-errors.zip](https://github.com/germanfr/cinnamon-transparent-panels/files/3827815/xsession-errors.zip)

**Describe the bug**
I recently updated to Cinnamon 4.3.0.  I am aware that as of this date of this bug report that Cinnamon 4.3.0 is not yet considered stable, but I thought that now would be a good time to let you know that your extension does not appear to work with it so that you can work on compatibility prior to the stable release.  Here is the error message that I get.

Extension transparent-panels@germanfr is not compatible with current version of cinnamon.  Using it may break your system.  Load anyway?

**Comment by germanfr** [comment]
Thanks for reporting :+1:

**Comment by Lantesh** [comment]
You are very welcome.  :-)

**Comment by germanfr** [comment]
Fixed in 01e4eec
